src/undo_log/undo_log.c

```diff
@@ -78,12 +99,12 @@ static void _UndoLog_Rollback_Update_Entity
 
 		// update indices
 		if(update_op->entity_type == GETYPE_NODE) {
-			Graph_UpdateEntity((GraphEntity *)&update_op->n, update_op->attr_id,
-				update_op->orig_value, update_op->entity_type);
+			_UndoLog_Restore_Entity_Property((GraphEntity *)&update_op->n, update_op->attr_id,
```

Comment:
in this case that we added new function in the undo log we can just sent the update_op as parameter no?

Reply:
I prefer to leave it this way as we will have to infer again the entity type